CMakeLists.txt

```diff
@@ -29,9 +29,9 @@ if(${PROJECT_SOURCE_DIR} STREQUAL ${PROJECT_BINARY_DIR})
     message(FATAL_ERROR "CMake generation is not allowed within source directory!! Please set a build folder with '-B'!!")
 endif()
 
-project(tt-metal
-        VERSION 1.0
-        DESCRIPTION "Tenstorrent Metallium"
+project(Metalium
+        VERSION 1.0.0
```

Comment:
Ya... lol we need to talk about how versioning is being done. Right now it's being ingested from a github action

Reply:
Sounds like current version is somewhere around 0.50-ish?  Let's hardcode this version here to `0.50` just so that our first cut at a package isn't going to be seen as something to upgrade to from better packages.

Reply:
Let's discuss offline.  Not necessary for this first step.